geonode/harvesting/harvesters/base.py

```diff
@@ -173,6 +174,17 @@ def update_geonode_resource(
                     f"the remote resource {harvested_info.resource_descriptor.uuid!r}")
             geonode_resource = resource_manager.update(
                 str(harvested_info.resource_descriptor.uuid), vals=defaults)
+
+        keywords = list(
+            harvested_info.resource_descriptor.identification.other_keywords
+        ) + geonode_resource.keyword_list()
+        keywords.append(
+            re.sub(' +', ' ', harvester.name.lower().replace('harvester', ''))
```

Comment:
I don't quite understand why you are replacing the string ` +` with ` ` here. Can you elaborate on this?

Reply:
hi @ricardogsilva 
it is for making double space to be single space in middle
for example, it it has name
"This is Harvester with Harvesting"
If we remove "harvester", it will be "This is  with "
Which is there is double space between "is" and "with"
it is regex to make every double space to be single space